ballerina/greeter/tests/service_test.bal

```diff
@@ -0,0 +1,35 @@
+import ballerina/io;
```

Comment:
Do we need this test file? 
I think we can remove it since this is a small sample.

Reply:
Let's keep it :) It's the boilerplate tests that come with the Ballerina greeter template

Reply:
If we run the test, it will fail, right? because the server is not running?

Reply:
`bal test` works!